src/util/web-util.js

```diff
@@ -52,14 +52,14 @@ function composePatchQuery (toDel, toIns) {
     toDel = toDel.map(function (st) {
       return statementToNT(st, excludeDot)
     })
-    query += 'DELETE DATA { ' + toDel.join(' . ') + ' };\n'
+    query += 'DELETE DATA { ' + toDel.join(' . ') + ' . };\n'
```

Comment:
So you're calling `excludeDot` just so you can append a dot later? What's the point of this code?